recipe/meta.yaml

```diff
@@ -234,12 +267,14 @@ outputs:
         - liblapack             # [blas_impl != "mkl"]
         - libgomp   # [linux]
         - llvm-openmp    # [osx]
+        # TODO(baszalmstra): There are linker errors on Windows when using protobuf from conda-forge
         - libabseil
         - libprotobuf
         - sleef
         - libuv
         - pkg-config  # [unix]
         - typing_extensions
+        - onnx 1.16  # [win]
```

Comment:
I really don't think this works.... basically from what I can tell, many people vendor onnx and just assume that this is the way life goes.

https://github.com/conda-forge/onnx-feedstock/pull/113

is my attempt at making a library out of it. but i don't think i ever managed to finish it.

Reply:
onnx for windows should be compiled in the same way as onnx for linux is compiled.

It seems the errors were abseil related. is abseil coming from the windows side?

Reply:
I don't think it's an abseil issue, is it? All that abseil provides is a warning that messages are written to stderr before initialising. The real problem seems to stem from protobuf, which states that "descriptor_database.cc:633] File already exists in database: onnx/onnx_onnx_torch-ml.proto"

There are some mentions in onnx issues: https://github.com/search?q=repo%3Aonnx%2Fonnx+%22File+already+exists+in+database%22&type=issues